pom.xml

```diff
@@ -62,11 +69,10 @@
       <version>3.15.6.Final</version>
     </dependency>
     <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
-      <version>4.13.2</version>
-      <scope>test</scope>
-    </dependency>
+			<groupId>org.junit.jupiter</groupId>
+			<artifactId>junit-jupiter</artifactId>
+			<scope>test</scope>
+		</dependency>
```

Comment:
These elements appear to be indented a good bit more than they should (if you don't see it, make sure you don't have "Hide whitespace" checked). I know it's kind of nit-picky, but could you fix the indentation so it's consistent with the rest?